fr/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md

```diff
@@ -0,0 +1,202 @@
+|             | SSHPASSWORD     | Ne peut pas être utilisé avec le backend. Seulement avec la clé d'authentication                |
+|             | SSHPORT         | Par default: 22                                                                                 |
+|             | SSHEXTRAOPTIONS | Personnalisez-le avec le vôtre si nécessaire. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```      |
+
+:warning: Avec ce backend, il est nécessaire d'effectuer une connexion manuelle entre l'utilisateur centreon-engine du Collecteur
+et l'utilisateur applicatif créé sur le serveur distant. (Macro SSHUSERNAME).
+
+<!--plink backend-->
+
+| Mandatory   | Name            | Description                                                                                     |
+| :---------- | :-------------- | :---------------------------------------------------------------------------------------------- | 
+| X           | SSHBACKEND      | Nom du backend: ```plink```                                                                     |
+| X           | SSHUSERNAME     | Par default, il utilise l'utilisateur en cours d'exécution ```centengine``` de votre Collecteur |
+|             | SSHPASSWORD     | Peut être utilisé. Si aucune valeur n'est définie, l'authentification par clé ssh est utilisée  |
+|             | SSHPORT         | Par default: 22                                                                                 |
+|             | SSHEXTRAOPTIONS | Personnalisez-le avec le vôtre si nécessaire. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```      |
+
+:warning: Avec ce backend, il est nécessaire d'effectuer une connexion manuelle entre l'utilisateur centreon-engine du Collecteur
+et l'utilisateur applicatif créé sur le serveur distant. (Macro SSHUSERNAME).
+
+<!--libssh backend (par défaut)-->
+
+| Mandatory   | Name            | Description                                                                                     |
+| :---------- | :-------------- | :---------------------------------------------------------------------------------------------- |
+| X           | SSHBACKEND      | Nom du backend: ```libssh```                                                                    |          
+|             | SSHUSERNAME     | Par default, il utilise l'utilisateur en cours d'exécution ```centengine``` de votre Collecteur |
+|             | SSHPASSWORD     | Peut être utilisé. Si aucune valeur n'est définie, l'authentification par clé ssh est utilisée  |
+|             | SSHPORT         | Par default: 22                                                                                 |
+|             | SSHEXTRAOPTIONS | Personnalisez-le avec le vôtre si nécessaire. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```      |
+
+Avec ce backend, vous n'avez pas à valider manuellement le fingerprint du serveur cible. 
+
+## FAQ
+
+### Comment puis-je tester le Plugin et que signifient les options des commandes ?
```

Comment:
Use second level instead of third (new convention)

Reply:
Changed